react-lists/my-app/src/App.tsx

```diff
@@ -0,0 +1,25 @@
+import './App.css';
+import { PokemonList } from './PokemonList';
+
+export type Pokemon = {
+  number: string;
+  name: string;
+};
+
+const pokedex: Pokemon[] = [
+  { number: '001', name: 'Bulbasaur' },
+  { number: '004', name: 'Charmander' },
+  { number: '007', name: 'Squirtle' },
+  { number: '025', name: 'Pikachu' },
+  { number: '039', name: 'Jigglypuff' },
+];
+
+function App() {
+  return (
+    <>
```

Comment:
Remove the fragments. They should only be there if they are required because your JSX has more than one parent.